src/midi_Message.h

```diff
@@ -114,6 +114,18 @@ struct Message
         const unsigned size = unsigned(data2) << 8 | data1;
         return size > sSysExMaxSize ? sSysExMaxSize : size;
     }
+    inline bool isSysRT () const
```

Comment:
Could we use fully qualified names (`isSystemRealTime` and `isSystemCommon`) for those methods please ?

Reply:
sure no problem.